app/context/ceramicContext.tsx

```diff
@@ -544,34 +544,43 @@ export const CeramicContextProvider = ({ children }: { children: any }) => {
     database: CeramicDatabase | PassportDatabase,
     skipLoadingState?: boolean
   ): Promise<void> => {
```

Comment:
Make sure not to load data from ceramic if fetchPassport is not called for the 1st time ....